Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/RoundImageEx.xaml

```diff
@@ -0,0 +1,111 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls">
+
+    <Style TargetType="controls:RoundImageEx">
+        <Setter Property="Background" Value="Transparent" />
+        <Setter Property="Foreground" Value="{ThemeResource ApplicationForegroundThemeBrush}" />
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="controls:RoundImageEx">
+                    <Grid>
+                        <Ellipse x:Name="PlaceholderEllipse"
```

Comment:
Its not difficult to create rounded image by tweaking corner radius. I am not sure if there's a default value that forces circular ellipse but my point is Border with RoundedCorner can be turned into fully circular image if desired. 
Ellipse can't do that. Any thoughts on why we should go for one or the other ?